Fails to compile with graphviz 3.0.0

If I'm trying to link with Graphviz 3.0.0 the linking fails with the following issue:

<details>
<summary>
Log output
</summary>

```
❯ python setup.py -v build_ext "-IC:\Users\<myusrename>\scoop\apps\graphviz\3.0.0\include" "-LC:\Users\<myusrename>\scoop\apps\graphviz\3.0.0\lib"
running build_ext
building 'pygraphviz._graphviz' extension
graphviz_wrap.c
pygraphviz/graphviz_wrap.c(3043): warning C4267: '=': conversion from 'size_t' to 'int', possible loss of data
pygraphviz/graphviz_wrap.c(3061): warning C4267: '=': conversion from 'size_t' to 'int', possible loss of data
Executing "C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.32.31326\bin\HostX86\x64\link.exe" /nologo /INCREMENTAL:NO /LTCG /DLL /MANIFEST:EMBED,ID=2 /MANIFESTUAC:NO /LIBPATH:C:\Users\<myusrename>\scoop\apps\graphviz\3.0.0\lib /LIBPATH:C:\<myvirtualenv>\.venv\libs /LIBPATH:C:\Users\<myusrename>\AppData\Local\Programs\Python\Python310\libs /LIBPATH:C:\Users\<myusrename>\AppData\Local\Programs\Python\Python310 /LIBPATH:C:\<myvirtualenv>\.venv\PCbuild\amd64 /LIBPATH:C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.32.31326\ATLMFC\lib\x64 /LIBPATH:C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.32.31326\lib\x64 /LIBPATH:C:\Program Files (x86)\Windows Kits\NETFXSDK\4.7.2\lib\um\x64 /LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.19041.0\ucrt\x64 /LIBPATH:C:\Program Files (x86)\Windows Kits\10\\lib\10.0.19041.0\\um\x64 cdt.lib cgraph.lib gvc.lib /EXPORT:PyInit__graphviz build\temp.win-amd64-3.10\Release\pygraphviz/graphviz_wrap.obj /OUT:build\lib.win-amd64-3.10\pygraphviz\_graphviz.cp310-win_amd64.pyd /IMPLIB:build\temp.win-amd64-3.10\Release\pygraphviz\_graphviz.cp310-win_amd64.lib
   Creating library build\temp.win-amd64-3.10\Release\pygraphviz\_graphviz.cp310-win_amd64.lib and object build\temp.win-amd64-3.10\Release\pygraphviz\_graphviz.cp310-win_amd64.exp
graphviz_wrap.obj : error LNK2001: unresolved external symbol Agdirected
graphviz_wrap.obj : error LNK2001: unresolved external symbol Agundirected
graphviz_wrap.obj : error LNK2001: unresolved external symbol Agstrictundirected
graphviz_wrap.obj : error LNK2001: unresolved external symbol Agstrictdirected
build\lib.win-amd64-3.10\pygraphviz\_graphviz.cp310-win_amd64.pyd : fatal error LNK1120: 4 unresolved externals
error: command 'C:\\Program Files\\Microsoft Visual Studio\\2022\\Community\\VC\\Tools\\MSVC\\14.32.31326\\bin\\HostX86\\x64\\link.exe' failed with exit code 1120
```
</details>

These commands work fine if I replace the previous call with:

```
❯ python setup.py -v build_ext "-IC:\Users\<myusrename>\scoop\apps\graphviz\2.49.3\include" "-LC:\Users\<myusrename>\scoop\apps\graphviz\2.49.3\lib"
```

--- a/setup.py
+++ b/setup.py
@@ -72,7 +72,7 @@ package_data = {
 if __name__ == "__main__":
     define_macros = []
     if sys.platform == "win32":
-        define_macros = define_macros.append(("GVDLL", None))
+        define_macros.append(("GVDLL", None))
 
     extension = [
         Extension(

